WcaOnRails/app/controllers/delegate_reports_controller.rb

```diff
@@ -29,6 +29,7 @@ def update
 
     @delegate_report = @competition.delegate_report
     @delegate_report.current_user = current_user
+    @delegate_report.discussion_url = "https://groups.google.com/forum/#!searchin/wca-delegates/subject$3A$20%22" + @delegate_report.competition.name.gsub(" ","$20") + "%22"
```

Comment:
1. This is pretty hacky, and will only work until Google changes some detail in their groups search. But it's better than nothing, or anything I can propose. So I guess it will have to do.

2. I think we have other special characters in competition names that need escaping aside from spaces. See https://github.com/cubing/worldcubeassociation.org/pull/689 for a list of allowed characters. Confusingly, this is a different encoding than the regular URL encoding, so URI::encode doesn't work here :(